vendor/engines/secure_rooms/spec/services/secure_rooms/access_handlers/order_handler_spec.rb

```diff
@@ -74,16 +74,22 @@
         describe "order_detail" do
           subject(:order_detail) { described_class.process(occupancy).order_details.first }
 
+          let(:price_group) { PriceGroup.first }
```

Comment:
I'm missing something conceptually here; I tried creating a `PriceGroup` in this let based on what attributes `PriceGroup.first` (which passes) has, but that didn't pass